packages/flutter_image/example/pubspec.yaml

```diff
@@ -17,7 +17,7 @@ dependencies:
     path: "../"
 
 dev_dependencies:
-  flutter_lints: ^2.0.0
+  flutter_lints: ^4.0.0
```

Comment:
This dependency is just a mistake, and should be removed. It's in the template and sometimes we forget to remove it.

Reply:
👍